src/Test.hs

```diff
@@ -214,14 +202,117 @@ durEx =    (translateX <:$> ((\x -> cos (x/2)) <:$> dur) <:*> (translateY <:$>
 ------------------
 -- cut
 cutEx :: A.Animation Double F Rasterific V2 Double
-cutEx = cut (1) movingRect
+cutEx = cut (3) solarSystem
+
+
+------------------
+-- solar system
+earth :: A.Animation Double I Rasterific V2 Double
+earth = ipure(circle 0.5 # fc blue)
+
+moon :: A.Animation Double I Rasterific V2 Double
+moon =    (translateX <:$> ((\x -> 0.7 * cos (3*x/2)) <:$> dur) <:*> (translateY <:$> 
+  ((\x -> 0.7 * sin (3*x/2))<:$> dur) <:*> (ipure (circle 0.1 # fc green # lc red))))
+
+moon2 :: A.Animation Double I Rasterific V2 Double
+moon2 =    (translateX <:$> ((\x -> 0.7 * cos (2*x)) <:$> dur) <:*> (translateY <:$> 
+  ((\x -> 0.7 * sin (2*x))<:$> dur) <:*> (ipure (circle 0.1 # fc green # lc red))))
+  
+moon3 :: A.Animation Double I Rasterific V2 Double
+moon3 =    (translateX <:$> ((\x -> 0.2 * cos (2*x)) <:$> dur) <:*> (translateY <:$> 
+  ((\x -> 0.2 * sin (2*x))<:$> dur) <:*> (ipure (circle 0.1 # fc green))))  
+
+mercury :: A.Animation Double I Rasterific V2 Double
+mercury = ipure(circle 0.25 # fc pink)
```

Comment:
All these definitions of planets is another opportunity for abstracting repeated code.